extensions-contrib/kubernetes-overlord-extensions/pom.xml

```diff
@@ -88,7 +88,7 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-common</artifactId>
-      <version>4.1.68.Final</version>
+      <version>4.1.86.Final</version>
```

Comment:
nit: Any reason not using: <netty4.version>?

Reply:
Fair point, updated the pom.